src/es/components/organisms/grid/Grid.js

```diff
@@ -12,87 +12,94 @@ import { Shadow } from '../../prototypes/Shadow.js'
  * @css {}
  */
 export default class Grid extends Shadow() {
-  connectedCallback () {
-    this.hidden = true
-    const showPromises = []
-    if (this.shouldComponentRenderCSS()) showPromises.push(this.renderCSS())
-    if (this.shouldComponentRenderHTML()) showPromises.push(this.renderHTML())
-    Promise.all(showPromises).then(() => (this.hidden = false))
+  async connectedCallback() {
+    this.hidden = true;
+    const showPromises = [];
+    if (this.shouldRenderCSS()) {
+      showPromises.push(this.renderCSS());
+    }
+    if (this.shouldRenderHTML()) {
+      showPromises.push(this.renderHTML());
+    }
+    await Promise.all(showPromises);
+    this.hidden = false;
   }
 
   /**
-   * evaluates if a render is necessary
+   * Evaluates whether a render is necessary.
    *
    * @return {boolean}
    */
-  shouldComponentRenderCSS () {
-    return !this.root.querySelector(`:host > style[_css], ${this.tagName} > style[_css]`)
+  shouldRenderCSS() {
+    return !this.root.querySelector(`:host > style[_css], ${this.tagName} > style[_css]`);
   }
 
   /**
-   * evaluates if a render is necessary
+   * Evaluates whether a render is necessary.
    *
    * @return {boolean}
    */
-  shouldComponentRenderHTML () {
-    return !this.grid
+  shouldRenderHTML() {
+    return !this.grid;
   }
 
   /**
-   * renders the o-grid css
+   * Renders the o-grid CSS.
    *
    * @return {Promise<void>}
    */
-  renderCSS () {
-    this.css = /* css */`
+  async renderCSS() {
+    const url = import.meta.url.replace(/(.*\/)(.*)$/, '$1');
```

Comment:
good catch! we have a scoped variable "this.importMetaUrl" on shadow, but it has to be fed with the import.meta.url through the constructor. this is part of the tide up sprint. see: https://github.com/mits-gossau/web-components-toolbox/blob/master/README.md#mater-branch-locked- transform all import.meta.url.replace(/(./)(.)$/, '$1') to this.importMetaUrl

Reply:
If okay, I would do it one by one.